src/components/ShortcutsModal.js

```diff
@@ -86,6 +86,14 @@ export class ShortcutsModal extends Component<Props> {
           L10N.getStr("shortcuts.functionSearch"),
           formatKeyShortcut(L10N.getStr("functionSearch.key"))
         )}
+        {this.renderShorcutItem(
+          L10N.getStr("shortcuts.gotoLine"),
+          formatKeyShortcut(L10N.getStr("gotoLineModal.key"))
+        )}
+        {this.renderShorcutItem(
```

Comment:
I think we can check if we're on linux. there is definitely a windows or mac check elsewhere